chloggen/internal/chlog/summary.go

```diff
@@ -18,57 +18,68 @@ import (
 	"bytes"
 	_ "embed"
 	"fmt"
-	"sort"
+	"path/filepath"
 	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
```

Comment:
Do we really need this? I'd rather stick to the standard templating package unless there is a clear need to bring in the dependency and expand the surface area.

Reply:
I thought it might be useful for custom templates. It's the same library helm uses (https://github.com/helm/helm/blob/0d0f91d1ce277b2c8766cdc4c7aa04dbafbf2503/pkg/engine/funcs.go#L26).

For the standard template we only need an `indent` function. I've reverted the dependency and implemented this function now (it's just a 2 liner), maybe we'll revisit this in the future if there is an actual user demand for more functions.

Reply:
Thanks @andreasgerstmayr. I'm _necessarily_ opposed to adding it later, just prefer not to add things until the need is clear.